data/json/items/ammo/8x40mm.json

```diff
@@ -0,0 +1,104 @@
+    "proportional": {
+      "price": 0.7,
+      "damage": 0.9,
+      "dispersion": 1.1
+    },
+    "extend": { "effects": [ "RECYCLED" ] },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40m",
+    "name_plural": "8x40m",
+    "//": "Cased ammo tends to be roughly $1/shot, more or less.  Rivtech ammo, being New and Proprietary and Expensive, $2-2.50 or so.",
+    "description": "8x40mm caseless rounds.  Proprietary ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "weight": 12,
+    "volume": 1,
+    "price": 12500,
+    "material": [ "plastic", "powder" ],
+    "symbol": "=",
+    "color": "pink",
+    "count": 40,
+    "stack_size": 40,
+    "ammo_type": "8x40mm",
+    "range": 42,
+    "damage": 42,
+    "pierce": 18,
+    "dispersion": 75,
+    "recoil": 400,
+    "effects": [ "NEVER_MISFIRES" ]
+  },
+  {
+    "id": "8mm_civilian",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40m sporting",
+    "name_plural": "8x40m sporting",
+    "description": "8x40mm caseless rounds, light sporting load sold for civilian use.  Proprietary ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "proportional": {
+      "price": 0.8,
+      "damage": 0.5,
+      "dispersion": 1.2,
+      "recoil": 0.5
+    },
+    "delete": { "effects": [ "NEVER_MISFIRES" ] }
+  },
+  {
+    "id": "8mm_fmj",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40mm FMJ",
+    "name_plural": "8x40mm FMJ",
+    "description": "8x40mm caseless rounds, full metal jacket.  Military grade ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "relative": {
+      "damage": -5,
+      "pierce": 10
+    }
+  },
+  {
+    "id": "8mm_hvp",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40mm HVP",
+    "name_plural": "8x40mm HVP",
+    "description": "8x40mm caseless rounds, saboted high-velocity penetrator type.  Advanced military grade, armor-piercing ammunition that fires a single depleted uranium flechette.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "price": 50000,
+    "count": 20,
+    "relative": {
+      "damage": 5,
+      "pierce": 20
+    }
+  },
+  {
+    "id": "8mm_inc",
+    "copy-from": "8mm_caseless",
+    "type": "AMMO",
+    "name": "8x40mm tracer",
+    "name_plural": "8x40mm tracer",
+    "description": "8x40mm caseless rounds with tracers to help keep the weapon they are fired from on target at the risk of igniting flammable substances.  Military grade ammunition for Rivtech firearms.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "count": 20,
+    "proportional": { "dispersion": 0.8 },
+    "extend": { "effects": [ "INCENDIARY" ] }
```

Comment:
Why are tracers strictly superior to non-tracers?
It's OK balance-wise, but doesn't make much sense.
Making tracers lower recoil significantly but have no effect on dispersion would model it better. Tracers help with re-targeting, but not sniping. Here they do the opposite.